src/wavm/openmp.cpp

```diff
@@ -388,17 +377,18 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
     Runtime::Memory* memoryPtr = parentModule->defaultMemory;
     faabric::Message* parentCall = getExecutingCall();
 
-    // Set up number of threads for next level
```

Comment:
Changes from here on are mostly to do with refactoring to the new Faabric set-up where we don't need to worry about spawning our own tasks etc.